src/test/kotlin/org/web3j/sokt/VersionResolverTest.kt

```diff
@@ -110,30 +110,30 @@ class VersionResolverTest {
         """
             0.5.13
```

Comment:
Changes here are only due to the fact that solc 0.4.26 was released since the tests were written & the assertion was technically the wrong way around.